### ISSUE TITLE
setting color from .(n)vimrc doesn't work

### ISSUE THREAD

**Issue by borring** [junegunn/vim-plug#254]
Have `Plug 'junegunn/seoul256.vim'` in your Plug section

Also add `color seoul256` to your .(n)vimrc. Restart (n)vim. The colorscheme seoul256 should be loaded. Instead, it says `E185: Cannot find color scheme 'seoul256'`

However, it works in the commandline with `:color seoul256` It also works from .(n)vimrc if I `cp plugged/seoul256.vim/colors/seoul256.vim colors`

**Comment by junegunn** [comment]
Did you put `:color seoul256` before `call plug#end()`? If that's not the case, please post your full vimrc.

**Comment by borring** [comment]
There's been a mistake. The first test _was_ done with `:color seoul256` before `plug#end()`. I then realized that it makes sense to move `:color ...` to somewhere after `plug#end()`. At the same time, I decided to change the color scheme before saving my nvimrc.

Coincidentally, the colorscheme that I chose has trouble with nvim true color support when launched from vimrc. So when I saw the messed up colors, I assumed it was the same error.

The colorscheme in question is `hybrid_material` btw. Almost the same symptoms, so I was confused. ::[This bug](https://github.com/neovim/neovim/issues/2953)
